bazel/src/main/java/com/google/api/codegen/bazel/Buildozer.java

```diff
@@ -92,7 +95,8 @@ public String getAttribute(Path bazelBuildFile, String target, String attribute)
   // immediately.
   public void setAttribute(Path bazelBuildFile, String target, String attribute, String value)
       throws IOException {
-    execute(bazelBuildFile, String.format("set %s \"%s\"", attribute, value), target);
+    final String escapedValue = value.replace(" ", "\\ ");
+    execute(bazelBuildFile, String.format("set %s \"%s\"", attribute, escapedValue), target);
```

Comment:
minor: just inlining replace call here would be just fine.

On a side note, it feels like `setAttribute()`, `addAttribute()`, `batchSetAttribute()`, `batchAddAttribute()` could become a one method, with an argument maybe.

Reply:
You know what, we don't need non-batch methods at all. We don't use them anywhere so YAGNI.  They are terribly slow since they execute `buildozer` for each invocation.